vue/src/components/Chart.vue

```diff
@@ -2,86 +2,159 @@
+    return {
+      data: {
+        labels: ['0', '5', '10', '15', '20', '25', '30'],
+        datasets: [
+          {
+            label: '人生グラフ',
+            backgroundColor: '#f565452',
+            data: [],
+            fillColor: 'rgba(165,42,42,0.65)', // 線から下端までを塗りつぶす色
+            strokeColor: 'rgba(165,42,42,1)', //  折れ線の色
+            pointColor: 'rgba(165,42,42,1)', //  ドットの塗りつぶし色
+            pointHighlightFill: 'yellow', // マウスが載った際のドットの塗りつぶし色/             pointHighlightStroke: 'yellow', // マウスが載った際のドットの枠線色
+            // label:ここにスコアが入るからlabelの中は空にする
+            borderWidth: 1,
+            fill: false,
+            lineTension: 0.4
+          }
+        ]
+      },
+      options: {
+        maintainAspectRatio: false,
+        tooltip: {
+          enabled: false,
+          custom: [] // 下で定義
+          // 定型文のfunction()はデータの中で使えないから下の方で出力
+        },
+        legend: false, // 凡例
+        scales: {
+          xAxes: [{
+            gridLines: {
+              //  縦線消す
+              display: false,
+              labelString: '年齢'
+            }
+          }],
+          yAxes: [{
+            gridLines: {
+              //  zerolineを残すため
+              display: false,
+              labelString: '満足度'
+              // zeroLineColor: 'rgba(86,84,82,1)'
+            },
+            ticks: {
+              beginAtZero: true,
+              max: 100,
+              min: -100,
+              stepsize: 10
+            }
+          }]
+        }
+      },
+      responsive: true
+    }
+  },
   mounted () {
-    this.renderChart({
-      labels: ['0', '5', '10', '15', '20', '25', '30'],
-      datasets: [
-        {
-          label: '人生グラフ',
-          backgroundColor: '#f565452',
-          data: [40, 20, 10, 80, 0, 60, 100],
-          fillColor: 'rgba(165,42,42,0.65)', // 線から下端までを塗りつぶす色
-          strokeColor: 'rgba(165,42,42,1)', //  折れ線の色
-          pointColor: 'rgba(165,42,42,1)', //  ドットの塗りつぶし色
-          pointHighlightFill: 'yellow', //  マウスが載った際のドットの塗りつぶし色/             pointHighlightStroke: 'yellow', // マウスが載った際のドットの枠線色
-          //  label:ここにスコアが入るからlabelの中は空にする
-          borderWidth: 1,
-          fill: false,
-          lineTension: 0.4
+    this.setAge()
+    this.setScore()
+    this.setComment()
+    this.renderChart(this.data, this.options)
+    // console.log(this.data)
+    // console.log(this.options)
+  },
+  methods: { // 処理を埋める
+    setAge () { // Age=.js age=vue
+      const age = []
+      this.$store.state.chart.contents.map((Age) => { // forEachじゃなくてmap使う
+        age.push(Age.age)
+      })
+      this.data.datasets[0].data = age // data()読まん
+    },
+    setScore () {
+      const score = []
+      this.$store.state.chart.contents.map((Score) => {
+        score.push(Score.score)
+      })
+      this.data.datasets[0].data = score
+    },
+    setComment () {
+      const comment = []
+      this.$store.state.chart.contents.map((Comment) => {
+        comment.push(Comment.comment)
+      })
```

Comment:
```suggestion
const comments = this.$store.state.chart.contents.map((contens) => {
  return { age: content.age, comment: content.comment }
});
```
こんな感じに書けるとシンプルかな
※あとでこの持たせ方にする必要がありそうなので、こういうデータの持たせ方をしてます。
該当の年齢に対するコメントを取得できるように。